Sdext.tex

```diff
@@ -149,6 +149,20 @@ \section{Reset}
 initialization that would usually happen before the first instruction is
 executed.
 
+\section{Halt}
+
+\begin{steps}{When a hart halts:}
+    \item \FcsrDcsrCause is updated.
+    \item \FcsrDcsrPrv is set to the current privilege mode.
+    \item \RcsrDpc is set to the next instruction that should be executed.
+    \item If the current instruction can be partially executed and should be
+        restarted to complete, then the relevant state for that is updated. E.g.
+        if a halt occurs during a partially executed vector instruction, then
+        {\tt vstart} is updated, and \RcsrDpc is updated to the address of the
+        partially executed instruction.
```

Comment:
Can we say something like "vstart is updated just like it would if it had any other exception (like a page fault) and dpc is updated just like mepc/sepc/vsepc would have been updated on any other exception (like a page fault)"?  (That needs some wordsmithing but I think it might be helpful for people to see that this basically uses existing mechanisms.)